client/src/App.js

```diff
@@ -150,8 +150,8 @@ class Document extends React.Component {
     }
     return (
       <div>
-        <div>
-          <h2>{document.title}</h2>
+        <div className="page-title">
```

Comment:
For: https://github.com/wbamberg/mdn2/blob/7b1e42cc44bdf01f1e1d0ef6fdf4dc7407124e6c/client/src/mdn.scss#L1-L4, where we add the grey background.

Reply:
Can we just edit the SCSS file to be just `h1` instead or something? It would be neat to keep the HTML/DOM as pure and simple as possible if there is no good reason for the extract `div`-pasta.

Reply:
Yes, that would work too.